### DISCUSSION TITLE
Aula - Acesso à dados com .NET

### DISCUSSION THREAD

**Discussion by mxlock** [balta-io/forum#357] [Tira Dúvidas] [answered]
Mesmo utilizando o código seguindo a aula. o código não me retornou nenhum valor de leitura com SqlCommand. [https://imgur.com/a/Q4cRMWe](url)

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @mxlock a imagem não veio, consegue anexar novamente?
Deu algum erro?

**Reply by mxlock** [reply]
No caso, não estou tendo resposta do banco de dados como saída no terminal, como na imagem.
Eu verifiquei a conexão com docker e login/senha também. 
https://imgur.com/gallery/nNX54lZ

**Reply by mxlock** [reply]
Possivelmente foi erro na hora da configuração do docker, vou refazer e testarei novamente.
